examples/common/tracing/TraceDecoder.h

```diff
@@ -42,7 +42,12 @@ class TraceDecoder : public TraceStream
     {
         char buffer[2048] = {};
         snprintf(buffer, sizeof(buffer), "    %s\t %s", tag.c_str(), data.c_str());
-        ReadString(buffer);
+        CHIP_ERROR err = ReadString(buffer);
```

Comment:
s/CHIP_ERROR/auto ?

Reply:
I think there is no large benefit between auto or CHIP_ERROR here and CHIP_ERROR seems slightly easier on human review (less thinking about type, even though err is also kind of obvious).

Reply:
That is fine by me.